feat(synthetics): Removed old IP and inserted new IP ranges, new call out, new metadescription (#6302)
newrelic/docs-website: src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx

---
title: Updated public minion IP address ranges
tags:
  - Synthetics
  - Synthetic monitoring
  - Administration
translate:
  - jp
metaDescription: 'New runtime capabilities for synthetic monitoring may require you update your firewall allow list to accept network requests from new IP address ranges'
---

> By the end of March 2022, an updated runtime will be launched to support newer versions of Chrome. As a result, you'll need to reconfigure your firewall's allow list, bot detection software, and other related applications to accept network requests from new IP address ranges. 

Reviewer: I want to point out that we tend to avoid "dating" changes and docs. It's a pain to keep dated info current and accurate.

In this case, I _think_ it's fine depending on the strategy here. Are customers aware this is the doc they should look at for this info? Will they be notified in some other form? I worry that just updating an already existing doc won't really serve as a good enough form of communication about these changes.

> By the end of March 2022, an updated runtime will be launched to support newer versions of Chrome. As a result, you'll need to reconfigure your firewall's allow list, bot detection software, and other related applications to accept network requests from new IP address ranges. 

Reviewer: See comment way down below, but I'd update the date as soon as you know the exact launch day.

> Monitors created after the new runtime launch will use the new IP address space. Existing monitors will be moved to the new IP address space later this year. Check out [Synthetic monitor public minion IPs](/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/) for more information about IP addresses and synthetic monitoring.

Reviewer: ```suggestion
Monitors created after the new runtime launch will use the new IP address space. Existing monitors will be moved to the new IP address space in the future. Check out [Synthetic monitor public minion IPs](/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/) for more information about IP addresses and synthetic monitoring.
```
